data/json/recipes/recipe_weapon.json

```diff
@@ -279,6 +279,44 @@
   ]
 },{
   "type" : "recipe",
+  "result": "spear_forked",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_PIERCING",
+  "skill_used": "fabrication",
+  "skills_required": [["survival", 2 ],["fabrication", 1]],
```

Comment:
OK.  Fab as the skill_used implicitly requires it at the difficulty.  No need to have it as a skills_required.